### ISSUE TITLE
docker built issue with builtins

### ISSUE THREAD

**Issue by pushingmyway** [tensorflow/serving#1371]
Hi，I follow the guide https://github.com/tensorflow/serving/blob/master/tensorflow_serving/g3doc/setup.md and run :

`tools/run_in_docker.sh bazel build -c opt tensorflow_serving/...`

then i meet the error:

```
== Pulling docker image: tensorflow/serving:nightly-devel
nightly-devel: Pulling from tensorflow/serving
Digest: sha256:c4d3edb7cef981e099de55fb416f0e1448675cfd6fcbda2ea399b47ce87d3376
Status: Image is up to date for tensorflow/serving:nightly-devel
== Running cmd: sh -c 'cd /home/work/indian/tmp3/serving; TEST_TMPDIR=.cache bazel build -c opt tensorflow_serving/...'
$TEST_TMPDIR defined: output root default is '/home/work/indian/tmp3/serving/.cache' and max_idle_secs default is '15'.
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
Traceback (most recent call last):
  File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py", line 29, in <module>
    from builtins import bytes  # pylint: disable=redefined-builtin
ImportError: No module named builtins
ERROR: /home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/core/BUILD:2779:1: no such package '@local_config_git//': Traceback (most recent call last):
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 61
		_fail(result.stderr)
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
		fail(("%sGit Configuration Error:%s %...)))
Git Configuration Error: Traceback (most recent call last):
  File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py", line 29, in <module>
    from builtins import bytes  # pylint: disable=redefined-builtin
ImportError: No module named builtins

 and referenced by '@org_tensorflow//tensorflow/core:version_info_gen'
ERROR: /home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/core/BUILD:2779:1: no such package '@local_config_git//': Traceback (most recent call last):
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 61
		_fail(result.stderr)
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
		fail(("%sGit Configuration Error:%s %...)))
Git Configuration Error: Traceback (most recent call last):
  File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py", line 29, in <module>
    from builtins import bytes  # pylint: disable=redefined-builtin
ImportError: No module named builtins

 and referenced by '@org_tensorflow//tensorflow/core:version_info_gen'
ERROR: /home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/core/BUILD:2779:1: no such package '@local_config_git//': Traceback (most recent call last):
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 61
		_fail(result.stderr)
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
		fail(("%sGit Configuration Error:%s %...)))
Git Configuration Error: Traceback (most recent call last):
  File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py", line 29, in <module>
    from builtins import bytes  # pylint: disable=redefined-builtin
ImportError: No module named builtins

 and referenced by '@org_tensorflow//tensorflow/core:version_info_gen'
ERROR: Analysis of target '//tensorflow_serving/servables/tensorflow/testdata:export_counter' failed; build aborted: no such package '@local_config_git//': Traceback (most recent call last):
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 61
		_fail(result.stderr)
	File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/third_party/git/git_configure.bzl", line 14, in _fail
		fail(("%sGit Configuration Error:%s %...)))
Git Configuration Error: Traceback (most recent call last):
  File "/home/work/indian/tmp3/serving/.cache/_bazel_work/218a8ab4f285f7991c962873081b5eb3/external/org_tensorflow/tensorflow/tools/git/gen_git_source.py", line 29, in <module>
    from builtins import bytes  # pylint: disable=redefined-builtin
ImportError: No module named builtins
```
i don't know whether the docker image missing package builtins?

**Comment by M1TR** [comment]
The missing _builtins_ module issue can be resolved by adding the "future" Python package in Dockerfile.devel, Dockerfile.devel-gpu, and Dockerfile.devel-mkl so that it looks like this:

`RUN pip --no-cache-dir install \
    grpcio \
    h5py \
    keras_applications \
    keras_preprocessing \
    mock \
    numpy \
    requests \
    future`

**Comment by netfs** [comment]
fixed as part of #1367. please re-open if this issue still persists.
